core/README.md

```diff
@@ -0,0 +1,160 @@
+# `@commercetools-test-data/core`
+
+This package provides the core functions to define the data models.
+
+## Install
+
+```bash
+$ yarn add -D @commercetools-test-data/core
+```
+
+## Creating a new data model
+
+Models are treated packages. Each model will be published as a standalone NPM package.
+
+To create a new model, add a new package in the `models` workspace with the following structure:
+
+```
+models/<package_name>
+├── LICENSE
+├── README.md
+├── index.ts
+├── package.json
+├── src
+│   ├── builder.ts
+│   ├── generator.ts
+│   ├── index.ts
+│   ├── presets
+│   │   └── index.ts
+│   ├── transformers.ts
+│   └── types.ts
+├── tsconfig.declarations.json
+└── tsconfig.json
+```
+
+To define and build new models, you need to use the core package `@commercetools-test-data/core`.
+
+### Builder
+
+A builder is the result of the data model. Builders allow composition with other models and to transform the data into different shapes, for example `rest` or `graphql`.
+
+When you want to get the generated data, you would **build** the model (builder).
+
+Builders are essentially composed by a generator and (optionally) transformers.
+
+### Generator
+
+A generator is what describes the initial model shape. Most of the fields can and should be initialized with some values. You can define random values using `fake` or static values.
+
+```ts
+const generator = Generator<TAuthor>({
+  name: 'Author',
+  fields: {
+    firstName: fake((f) => f.name.firstName()),
+    locale: 'en',
+  },
+});
+```
```

Comment:
```suggestion
```ts
import { fake } from '@jackfranklin/test-data-bot';

const generator = Generator<TAuthor>({
  name: 'Author',
  fields: {
    firstName: fake((f) => f.name.firstName()),
    locale: 'en',
  },
});
```
```

Reply:
The `fake` is exported from our core package, so we should import from that. Thanks for the suggestion.

Reply:
Yes I noticed that lately, the idea is to document the import.